src/core/output-layout.cpp

```diff
@@ -665,7 +665,14 @@ struct output_layout_output_t
         }
 
         wlr_dmabuf_attributes attributes;
-        if (!wlr_output_export_dmabuf(wo->handle, &attributes))
+        if (wo->handle->front_buffer == NULL)
+        {
+            LOGE("Got empty buffer on ", wo->handle);
```

Comment:
I am not sure why we're printing a pointer to the `wlr_output` here. It might make more sense to output `wo->to_string()` or `wo->handle->name`, this way, you get a human-readable identification of the output where the error happens.